cluster/userdata-master.yaml

```diff
@@ -853,6 +853,51 @@ write_files:
               securityContext:
                 privileged: true
 
+
+  - path: /srv/kubernetes/manifests/deployments/cluster-autoscaler.yaml
+    content: |
+      apiVersion: extensions/v1beta1
+      kind: Deployment
+      metadata:
+        name: cluster-autoscaler
+        namespace: kube-system
+        labels:
+          app: cluster-autoscaler
+          version: v1
+      spec:
+        replicas: 1
+        selector:
+          matchLabels:
+            app: cluster-autoscaler
```

Comment:
You don't need to define the `matchLabels` because it's automatically derived from the `template.labels` below.